application/jaxrs-hazelcast-quarkus/src/main/java/webapp/tier/service/HazelcastMessageListener.java

```diff
@@ -0,0 +1,33 @@
+package webapp.tier.service;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import com.hazelcast.topic.Message;
+import com.hazelcast.topic.MessageListener;
+
+import webapp.tier.bean.MsgBean;
+import webapp.tier.service.socket.HazelcastSocket;
+import webapp.tier.util.MsgUtils;
+
+@ApplicationScoped
+public class HazelcastMessageListener implements MessageListener<Object> {
+
+	private static final Logger LOG = Logger.getLogger(HazelcastMessageListener.class.getSimpleName());
+	private static String splitkey = ConfigProvider.getConfig().getValue("hazelcast.split.key", String.class);
+	HazelcastSocket hazsock = new HazelcastSocket();
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6010272)